AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/ModelSchema+SQLite.swift

```diff
@@ -119,7 +119,7 @@ extension ModelSchema {
     func createIndexStatements() -> String {
         var statement = ""
         for index in indexes {
-            if case let .index(fields, name) = index, let name = name {
+            if case let .index(fields, name?) = index {
```

Comment:
Does this make `name` an optional whereas earlier it was required to have a value?

Reply:
Nope, this only executes if `name != nil`.

Reply:
@thisisabhash may have some context here

Reply:
Nope, `CreateIndexStatement(modelSchema:fields:indexName:)` takes a non-optional `String`. The trailing `?` makes use of the [Optional Pattern Matching Operator](https://docs.swift.org/swift-book/ReferenceManual/Patterns.html#ID520) to safely unwrap and only execute if `name != nil`